### PR TITLE
Support exporting current container to image

### PR DESCRIPTION
**Pull request by JaewonHur**
This PR adds a command to export container to image (#1019).

## Type of Change
- [ ] Bug fix
- [X] New feature  
- [ ] Breaking change
- [ ] Documentation update

## Motivation and Context
Users can export their container to image.

## Testing
- [X] Tested locally
- [ ] Added/updated tests
- [ ] Added/updated docs

### CODE REVIEW

File: Sources/ContainerCommands/ExportCommand.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import Foundation
+import TerminalProgress
+
+extension Application {
+    public struct ExportCommand: AsyncLoggableCommand {
+        public init() {}
+        public static var configuration: CommandConfiguration {
+            var config = CommandConfiguration()
+            config.commandName = "export"
+            config.abstract = ""
```
Comment (ajemory) [inline Sources/ContainerCommands/ExportCommand.swift]: Add a help message to .abstract

File: Sources/ContainerCommands/ExportCommand.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import ContainerAPIClient
+import ContainerizationError
+import Foundation
+import TerminalProgress
+
+extension Application {
+    public struct ExportCommand: AsyncLoggableCommand {
+        public init() {}
+        public static var configuration: CommandConfiguration {
+            var config = CommandConfiguration()
```
Comment (ajemory) [inline Sources/ContainerCommands/ExportCommand.swift]: nit: it's cleaner to pass the config options as parameters here instead of initializing config and setting them after

File: Sources/ContainerCommands/ExportCommand.swift [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            config.commandName = "export"
+            config.abstract = ""
+            config._superCommandName = "container"
```
Comment (ajemory) [inline Sources/ContainerCommands/ExportCommand.swift]: I would move this entire file into ContainerCommands/Container, rename it ContainerExport and remove _superCommandName (Moving it to Image instead of Container could also make sense). This will also set helpNames to -h and --help by default, so you can remove that as well

**Comment by JaewonHur** [comment]
@jglogan This PR has minimal test writing a file before exporting a container and checking it remains correct in the exported image.
Do you have any edge cases to be checked ? e.g., volume mount should not be preserved..?

**Review by ajemory — APPROVED** [review]
